src/vllm_tgis_adapter/grpc/pb/generation.proto

```diff
@@ -225,6 +225,9 @@ message TokenInfo {
 
 message BatchedTokenizeRequest {
   string model_id = 1;
+  // Deprecated in favor of adapter_id
+  optional string prefix_id = 5;
```

Comment:
We cannot use `5` here as it's used by `truncate_input_tokens` below:

```
      vllm_tgis_adapter/grpc/pb/generation.proto:236:34: Field number 5 has already been used in "fmaas.BatchedTokenizeRequest" by field "prefix_id".
```

Reply:
Thanks @dtrifiro my bad, pushed a fix now.